src/Modules/Admin/Commands.Admin/help/MicrosoftPowerBIMgmt.Admin.md

```diff
@@ -0,0 +1,25 @@
+---
+Module Name: MicrosoftPowerBIMgmt.Admin
+Module Guid: bbf8be8e-0dcd-4cd8-b868-63e96371a47f
+Download Help Link: {{Please enter FwLink manually}}
+Help Version: {{Please enter version of help manually (X.X.X.X) format}}
+Locale: en-US
+---
+
+# MicrosoftPowerBIMgmt.Admin Module
+## Description
+PowerShell cmdlets for managing and interacting with Power BI admin services.
```

Comment:
admin -> administrative
I would word this slightly different: PowerShell cmdlets for performing administrative duties the across Power BI service.